From e8534b500525935ea7559f07cb8cc30854188211 Mon Sep 17 00:00:00 2001
Message-ID: <e8534b500525935ea7559f07cb8cc30854188211.1701297273.git.sharlatanus@gmail.com>
In-Reply-To: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
References: <123a0b9b76c24c29184c527dabb652384f7be762.1701297273.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Sat, 28 Oct 2023 03:07:21 +0100
Subject: [PATCH v3 06/66] gnu: go-github-com-puerkitobio-goquery: Move to (gnu
 packages golang-web).

* gnu/packages/golang.scm (go-github-com-puerkitobio-goquery): Move from
here...
* gnu/packages/golang-web.scm: ...to here.

Change-Id: Ic0415a8b280ebd12086a5976c9765c1dbbf50bd2
---
 gnu/packages/golang-web.scm | 26 ++++++++++++++++++++++++++
 gnu/packages/golang.scm     | 24 ------------------------
 2 files changed, 26 insertions(+), 24 deletions(-)

diff --git a/gnu/packages/golang-web.scm b/gnu/packages/golang-web.scm
index 3c1abfff2c..8d8b4fb85c 100644
--- a/gnu/packages/golang-web.scm
+++ b/gnu/packages/golang-web.scm
@@ -2,6 +2,8 @@
 ;;; Copyright © 2018 Pierre-Antoine Rouby <pierre-antoine.rouby@inria.fr>
 ;;; Copyright © 2020 Martin Becze <mjbecze@riseup.net>
 ;;; Copyright © 2020 Oleg Pykhalov <go.wigust@gmail.com>
+;;; Copyright © 2021 Efraim Flashner <efraim@flashner.co.il>
+;;; Copyright © 2021 Sarah Morgensen <iskarian@mgsn.dev>
 ;;; Copyright © 2022 Giacomo Leidi <goodoldpaul@autistici.org>
 ;;;
 ;;; This file is part of GNU Guix.
@@ -131,6 +133,30 @@ (define-public go-github-com-gorilla-mux
 incoming requests with their respective handler.")
     (license license:bsd-3)))
 
+(define-public go-github-com-puerkitobio-goquery
+  (package
+    (name "go-github-com-puerkitobio-goquery")
+    (version "1.7.0")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/PuerkitoBio/goquery")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "0gh1d99l5xc9hvwa4j40pfq3y9vfyq52mnrz6bf1kw2r2zr2gbcc"))))
+    (build-system go-build-system)
+    (arguments
+     `(#:import-path "github.com/PuerkitoBio/goquery"))
+    (propagated-inputs
+     (list go-github-com-andybalholm-cascadia go-golang-org-x-net))
+    (home-page "https://github.com/PuerkitoBio/goquery")
+    (synopsis "Features similar to jQuery to the Go language")
+    (description "@code{goquery} brings a syntax and a set of features similar
+to jQuery to the Go language.")
+    (license license:bsd-3)))
+
 ;;;
 ;;; Avoid adding new packages to the end of this file. To reduce the chances
 ;;; of a merge conflict, place them above by existing packages with similar
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index a6365a9686..7868f71206 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -9330,30 +9330,6 @@ (define-public go-github-com-alecthomas-chroma
 converts it into syntax highlighted HTML, ANSI-coloured text, etc.")
     (license license:expat)))
 
-(define-public go-github-com-puerkitobio-goquery
-  (package
-    (name "go-github-com-puerkitobio-goquery")
-    (version "1.7.0")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/PuerkitoBio/goquery")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "0gh1d99l5xc9hvwa4j40pfq3y9vfyq52mnrz6bf1kw2r2zr2gbcc"))))
-    (build-system go-build-system)
-    (arguments
-     `(#:import-path "github.com/PuerkitoBio/goquery"))
-    (propagated-inputs
-     (list go-github-com-andybalholm-cascadia go-golang-org-x-net))
-    (home-page "https://github.com/PuerkitoBio/goquery")
-    (synopsis "Features similar to jQuery to the Go language")
-    (description "@code{goquery} brings a syntax and a set of features similar
-to jQuery to the Go language.")
-    (license license:bsd-3)))
-
 (define-public go-github-com-jmespath-go-jmespath
   (package
     (name "go-github-com-jmespath-go-jmespath")
-- 
2.41.0

